traffic_ops/traffic_ops_golang/traffic_ops_golang.go

```diff
@@ -61,11 +63,31 @@ func main() {
 	}
 
 	if err := log.InitCfg(cfg); err != nil {
-		fmt.Println("Error initializing loggers: %v", err)
+		fmt.Printf("Error initializing loggers: %v\n", err)
 		return
 	}
 
-	log.Infof("Using Config: %+v\n", cfg)
+	log.Infof(`Using Config values:
+		Port:                %s
+		Db Server:           %s
+		Db User:             %s
+		Db Name:             %s
+		Db Ssl:              %t
+		Max Db Connections:  %d
+		TO URL:              %s
+		Insecure:            %t
+		Cert Path:           %s
+		Key Path:            %s
+		Proxy Timeout:       %v
+		Read Timeout:        %v
+		Read Header Timeout: %v
+		Write Timeout:       %v
+		Idle Timeout:        %v
+		Error Log:           %s
+		Warn Log:            %s
+		Info Log:            %s
+		Debug Log:           %s
+		Event Log:           %s`, cfg.HTTPPort, cfg.DBServer, cfg.DBUser, cfg.DBDB, cfg.DBSSL, cfg.MaxDBConnections, cfg.TOURLStr, cfg.Insecure, cfg.CertPath, cfg.KeyPath, time.Duration(cfg.ProxyTimeout) * time.Second, time.Duration(cfg.ReadTimeout) * time.Second, time.Duration(cfg.ReadHeaderTimeout) * time.Second, time.Duration(cfg.WriteTimeout) * time.Second, time.Duration(cfg.IdleTimeout) * time.Second, cfg.LogLocationError, cfg.LogLocationWarning, cfg.LogLocationInfo, cfg.LogLocationDebug, cfg.LogLocationEvent)
```

Comment:
What about using a nested struct instead, to hide sensitive data? It's a lot less code, and people don't have to remember to add things here too.

```
type ConfigOpen struct {
 Foo int
 Bar string
}
type Config struct {
  ConfigOpen
  Secret string
  Pass string
}
```
```
	cfg := Config {
		ConfigOpen: ConfigOpen{
			Foo: 42,
			Bar: "baz",
		},
		Secret: "walrus",
		Pass: "carpenter",
	}
	fmt.Printf("Config: %+v\n", cfg.ConfigOpen)
```

https://play.golang.org/p/iO2T96zIQI

Reply:
I like this idea and will see how to make it work smoothly, becuase yes, updating each time a field is added has been a pain point in this PR already.